opentelekomcloud/config.go

```diff
@@ -8,6 +8,8 @@ import (
 	"net/http"
 	"os"
 
+	"strings"
```

Comment:
Got some unnecessary whitespace between `os` and `strings`

Reply:
Done